libraries/abstractions/ble_hal/test/src/iot_test_ble_hal_integration.c

```diff
@@ -299,42 +299,58 @@ TEST_GROUP_RUNNER( Full_BLE_Integration_Test )
     }
 #endif /* if ENABLE_TC_INTEGRATION_ADVERTISE_WITH_16BIT_SERVICEUUID */
 
-#if ENABLE_TC_INTEGRATION_ADVERTISE_WITH_MANUFACTUREDATA
-    TEST( Full_BLE_Integration_Test_Advertisement, BLE_Advertise_With_ManufactureData )
+#if ENABLE_TC_INTEGRATION_ADVERTISE_WITH_MANUDATA_SERVDATA
+    TEST( Full_BLE_Integration_Test_Advertisement, BLE_Advertise_With_ManuData_ServData )
     {
         /* Manufacturer-specific Data
          * First two bytes are company ID (randomly select Espressif(0x02E5) for test purpose)
-         * Next bytes are defined by the company (randomly select unit8_t 5 for test purpose)*/
+         * Next bytes are defined by the company (randomly select 0x05 for test purpose)*/
+
+        /* Service Data,
+         * First two bytes are 16bit Service UUID (randomly select 0xEF12 for test purpose)
+         * Next bytes are Service Data (randomly select 0xD6 for test purpose)*/
         uint16_t usManufacturerLen = 3;
         uint8_t pusManufacturerData[] = { 0xE5, 0x02, 0x05 };
```

Comment:
Value needs to be defined as a `#define`  so that it is easy to check values at both sides (RPI and DUT)